### PR TITLE
migration: add callout for rescheduling mechanism

### PR DESCRIPTION
**Pull request by ryancharris**
## What are you changing in this pull request and why?
recently, the multicell team added a component to the in-app migration guide which allows customers to postpone their account migration by up to 45 days (this functionality is currently only available to enterprise tier accounts).

as we begin migrating more enterprise accounts, we want our public facing docs to highlight this for our users.

## Checklist
- [x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).

### CODE REVIEW

File: website/docs/docs/cloud/migration.md [hunk 1]
```diff
@@ -15,6 +15,13 @@ Your account will be automatically migrated on or after its scheduled date. Howe
 
 ## Recommended actions
 
+:::info Rescheduling your migration
+
+Customers on dbt Cloud Enterprise tiers can postpone their account migration by up to 45 days. To reschedule your migration, go to **Account Settings** → **Migration guide**.
+
+For help, contact us at [sales@getdbt.com](mailto:sales@getdbt.com).
```
Comment (aaronraff) [inline website/docs/docs/cloud/migration.md]: Should this be our support email instead, or is sales appropriate for Enterprise accounts?

Comment (ryancharris) [reply website/docs/docs/cloud/migration.md]: @aaronraff I believe this should be `support@getdbt.com`.

**Review by aaronraff — APPROVED** [review]

**Review by eiof — APPROVED** [review]

File: website/docs/docs/cloud/migration.md [hunk 2]
```diff
@@ -15,6 +15,13 @@ Your account will be automatically migrated on or after its scheduled date. Howe
 
 ## Recommended actions
 
+:::info Rescheduling your migration
+
+Customers on dbt Cloud Enterprise tiers can postpone their account migration by up to 45 days. To reschedule your migration, go to **Account Settings** → **Migration guide**.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/migration.md]: Just to keep the language consistent. We generally use 2nd person.
```suggestion
If you're on the dbt Cloud Enterprise tier, you can postpone your account migration by up to 45 days. To reschedule your migration, navigate to **Account Settings** → **Migration guide**.
```

File: website/docs/docs/cloud/migration.md [hunk 3]
```diff
@@ -15,6 +15,13 @@ Your account will be automatically migrated on or after its scheduled date. Howe
...
+
+For help, contact us at [support@getdbt.com](mailto:support@getdbt.com).
```
Comment (matthewshaver) [inline website/docs/docs/cloud/migration.md]:
```suggestion
For help, contact the dbt Support Team at [support@getdbt.com](mailto:support@getdbt.com).
```

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**